.github/workflows/golangci-lint.yml

```diff
@@ -14,10 +14,6 @@ name: golangci-lint
 on:
   pull_request:
     branches: main
-    types:
-      - opened
-      - reopened
-      - synchronize
```

Comment:
I would recommend reverting this, otherwise it will run any time there is any kind of update to a PR (e.g. title changes, it gets merged, someone comments, etc.)

Reply:
That does not happen, but i will revert

Reply:
Yep these would only trigger on a push to the branch and when the PR is (re)opened.

Reply:
Normally, it will trigger when you open/reopen/sync. If you want to trigger other actions, then you set it.

so that is redundant to set as those are the default triggers

Reply:
@cpanato Huh I didn't know that, they really didn't make it easy to find. Learn something new every day!

From the first paragraph under the notes [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request): 

> Runs your workflow when activity on a pull request in the workflow's repository occurs. For example, if no activity types are specified, the workflow runs when a pull request is opened or reopened or when the head branch of the pull request is updated.

Then again, to quote the Zen of Python, explicit is better than implicit 😄